kola/tests/ignition/mount.go

```diff
@@ -0,0 +1,411 @@
+// Copyright 2019 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ignition
+
+import (
+	"strings"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform/conf"
+)
+
+func init() {
+	// verify mount to `/var/lib/containers`
+	register.Register(&register.Test{
+		Name:        "coreos.ignition.mount.var-lib-containers",
+		Run:         mountVarLibContainers,
+		ClusterSize: 1,
+		UserData: conf.Ignition(`{
+			"ignition": {"version": "2.2.0"},
```

Comment:
Before we add many more tests that use nontrivial Ignition, we're really going to need to migrate the Ignition translation from cosa to here, and only write it one time.  @ajeddeloh may have newer code.

Reply:
:+1 on this, this would be very helpful
So for now maybe only test ignition v3 on FCOS?

Reply:
@ajeddeloh Do you indeed have code ready to go we could use for this?

I think we can just hackily have v2 and v3 Ignition for now so we can get this test in and circle back to deduping once we have a solid translator in place. OTOH, if there's code ready to go to accomplish this, then let's definitely leverage that.